blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -247,7 +248,7 @@ private static void startEventListenerModules() {
         }
     }
 
-    public static void main(String args[]) {
+    public static void main(String args[]) throws BluefloodServiceStarterException {
```

Comment:
I dont think we should throw exception out of main method. We should catch it in the main method and exit with a proper exit code.